### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ghost974**
Add AWS permissions to find and attach an EBS volume to an EC2 Instance.

### CODE REVIEW

**Comment by ghost974** [comment]
I signed it!

**Comment by justinsb** [comment]
@ghost974 Thanks for the change.  If you use the cluster/kube-up script, you should get these permissions anyway (https://github.com/GoogleCloudPlatform/kubernetes/blob/master/cluster/aws/templates/iam/kubernetes-minion-policy.json).

Are you not getting them (i.e. is there a bug in the script?).

Or is this PR just documenting that these are the permissions required (if so, I think we should say that cluster/kube-up will do this for you, so that people don't think this is much harder than it really is!)

**Comment by ghost974** [comment]
I started my cluster with this guide : https://github.com/GoogleCloudPlatform/kubernetes/blob/master/docs/getting-started-guides/aws-coreos.md
So I didn't use any of the script kube-*, that's why I didn't have the proper permission.

**Comment by wojtek-t** [comment]
@ghost974 - should we proceed with this PR? If so can you please rebase?

**Comment by mbforbes** [comment]
(Still needs rebase)

File: examples/aws_ebs/README.md [hunk 1]
```diff
@@ -1,9 +1,16 @@
...
-the pod:
+First make sure that your minions have the following IAM permissions : (http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/iam-roles-for-amazon-ec2.html#create-iam-role-console):
+```shell
+  ec2:AttachVolume
+  ec2:DetachVolume
+  ec2:DescribeInstances
+  ec2:DescribeVolumes
+```
+
+Create a volume in the same region as your node.<br>
```
Comment (mbforbes) [inline examples/aws_ebs/README.md]: replace `<br> ` with just an empty line

File: examples/aws_ebs/README.md [hunk 2]
```diff
@@ -1,9 +1,16 @@
 This is a simple web server pod which serves HTML from an AWS EBS
 volume.
 
-Create a volume in the same region as your node add your volume
-information in the pod description file aws-ebs-web.yaml then create
-the pod:
+First make sure that your minions have the following IAM permissions : (http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/iam-roles-for-amazon-ec2.html#create-iam-role-console):
+```shell
```
Comment (mbforbes) [inline examples/aws_ebs/README.md]: I prefer an empty line before big block quoted sections, but this is your call.

File: examples/aws_ebs/README.md [hunk 3]
```diff
@@ -1,9 +1,16 @@
 This is a simple web server pod which serves HTML from an AWS EBS
 volume.
 
-Create a volume in the same region as your node add your volume
-information in the pod description file aws-ebs-web.yaml then create
-the pod:
+First make sure that your minions have the following IAM permissions : (http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/iam-roles-for-amazon-ec2.html#create-iam-role-console):
```
Comment (mbforbes) [inline examples/aws_ebs/README.md]: Could you make a descriptive name for the page you've linked to and then make this just a markdown link?

Example: `[text describing linked page](url#section)`

**Comment by mbforbes** [comment]
Based on https://github.com/kubernetes/kubernetes/pull/10174#issuecomment-118894757 I think you should mention that the kube-up script does this for those not using the same guide you did

**Comment by mbforbes** [comment]
Friendly ping @ghost974, this PR needs a rebase if you're still interested in getting it in!

**Comment by mbforbes** [comment]
I'm unassigning myself because I'm leaving the project. @ghost974 if you successfully rebase, please ping the Kubernetes oncall to assign someone to take a look!

**Comment by wojtek-t** [comment]
@ghost974 is this PR still active? If so can you please rebase?

**Comment by ghost974** [comment]
@wojtek-t let me know if it's good, first time I work with different repos at the same time, not sure if I did it right !

**Comment by wojtek-t** [comment]
LGTM

**Comment by wojtek-t** [comment]
@thockin - can you please take a look?

**Comment by karlkfi** [comment]
heh, love flakey tests on readme changes.. so obvious.
Rerunning...

**Comment by kelseyhightower** [comment]
LGTM

**Comment by eparis** [comment]
@k8s-bot test this

**Comment by wojtek-t** [comment]
@k8s-bot unit test this please

**Pull request merged**